locallibrary/settings.py

```diff
@@ -0,0 +1,135 @@
+"""
+Django settings for locallibrary project.
+
+Generated by 'django-admin startproject' using Django 3.2.20.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/3.2/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/3.2/ref/settings/
+"""
+
+from pathlib import Path
+from decouple import config
```

Comment:
dùng module `python-dotenv` cho phần config biến môi trg nhé
https://dev.to/earthcomfy/django-how-to-keep-secrets-safe-with-python-dotenv-5811